pkg/minishift/openshift/oc.go

```diff
@@ -42,3 +47,38 @@ func AddSudoersRoleForUser(user string) error {
 	}
 	return nil
 }
+
+// Get the route for service
+func GetServiceURL(service, namespace string) (string, error) {
+	cmdName := instanceState.Config.OcPath
+	cmdArgText := fmt.Sprintf("get route/%s", service)
+	if namespace != "" && !isProjectExists(namespace) {
+		return "", errors.New(fmt.Sprintf("Namespace %s doesn't exits", namespace))
+	}
+	if namespace != "" {
```

Comment:
@gbraad here empty namespace is also a valid case.

Eg

minishift openshift service <name>     will use current namespace (which is default to empty string).
However, if user want to open app from different namespace/project then he need to specify namespace then this `if` statement check that and add `-n namespace` to command.